pygeoapi-config.yml

```diff
@@ -302,3 +302,4 @@ resources:
         type: process
```

Comment:
Are changes to this file necessary?

Reply:
It's as a result of syncing with the master branch.

Reply:
Can we remove this addition?